paf-mvp-demo-express/src/views/advertiser/index.hbs

```diff
@@ -820,11 +820,11 @@
                   </span>
                 </a>
                 <script>
-                  document.querySelector('#preferences')
-                    .addEventListener('click', (e) => {
-                      e.preventDefault();
-                      PAF.refreshIdsAndPreferences({ proxyHostName: "{{proxyHostName}}", showPrompt: 'doPrompt'});
-                    });
+                    document.querySelector('#preferences')
+                        .addEventListener('click', (e) => {
+                            e.preventDefault();
+                            window.PAFUI.promptConsent();
```

Comment:
Actually, no, you shouldn't do this.
Seeing your change, I investigated why there was a difference between the publisher and the advertiser, and there shouldn't be.
More problematic: it happens that this "new" version (which is used on the publisher's view) has a bug.

You can test it yourself:

- access [publisher](https://www.pifdemopublisher.com/)
- the widget pops up
- close it
- click “edit marketing preferences”
- the original widget is shown (no possibility to edit id)
- click any of the two options
- nothing is saved, the user is still NOT_PARTICIPATING
- Expected behavior: should save an appropriate cookies

Reply:
The bug with the writing of the data will be addressed in a following on PR covering all the fixes needed to logic. Resolving as not in the scope of this PR which is about languages and separate bundles for each one.

Reply:
I'm not sure we are talking about the same thing. I'm talking about an existing bug (independent from your PR) that was fixed here: https://github.com/prebid/paf-mvp-implementation/pull/156

This is why I wouldn't want you to create a regression here.

Reply:
I've merged 156 and also updated the advertiser footer to match the footer in the publisher.